projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java

```diff
@@ -253,7 +253,9 @@ public DataJobExecution readJobExecution(String teamName, String jobName, String
                   com.vmware.taurus.service.model.DataJobExecution.builder()
                         .id(jobExecution.getExecutionId())
                         .dataJob(dataJob)
-                        .startTime(jobExecution.getStartTime())
+                        .startTime(jobExecution.getStartTime() != null ?
```

Comment:
Can you add a test in JobExecutionServiceUpdateExecutionIT which covers the startTime initialization?

Reply:
Will do.